src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -487,23 +535,46 @@ _matrix_for_polymake(::Val{_hilbert_generator}) = _generator_matrix
 
 
 @doc raw"""
-    contains(C::Cone, v::AbstractVector)
+    in(C0::Cone, C1::Cone)                           
+    
+Check whether `C0` is contained in `C1` as a subset.
```

Comment:
A subset is not an element of a set, though. So this does not seem like an appropriate `in` method? Rather sounds like it should be a method for `issubset`?

Reply:
That would also remove the need to write "as a subset" in the docstring, it would be obvious from the function name

Reply:
https://github.com/oscar-system/Oscar.jl/pull/2230#discussion_r1161150807

Sorry, I think I misunderstood the above comment, probably you also selected the wrong block?

Reply:
No worries. I've spent easter with fever and am slowly recovering right now so I probably am not very coherent:-/.